### PR TITLE
implement MLflow Model wrappper for DAI MOJO

### PR DESCRIPTION
**Pull request by MartinBarus**
This PR implements simple command line tool that creates MLflow Model wrapper of Driverless AI MOJO model that can then be used by mlflow to be deployed in:

- Azure ML
- Amazon SageMaker
- Local REST Server
- Apache Spark UDF

https://www.mlflow.org/docs/latest/models.html

### CODE REVIEW

File: mlflow/example/license.sig [hunk 1]
```diff
@@ -0,0 +1 @@
+ENTER VALID DRIVERLESS AI LICENSE
```
Comment (nkpng2k) [inline mlflow/example/license.sig:1]: nit: do we need this file at all? I would expect based on the scoring code below + readme that this should be understood, but I am fine with keeping it.

File: mlflow/model/dai_model.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    mlflow.pyfunc.save_model(
+        path=model_name,
+        python_model=DAIWrappedModel(),
+        artifacts=artifacts,
+        conda_env=conda_env,
+    )
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
```
Comment (nkpng2k) [inline mlflow/model/dai_model.py:81]: nit: I'd generally advocate for making this a method, but that doesn't really change the functionality.

Comment (nkpng2k) [reply mlflow/model/dai_model.py:81]: or maybe just define full class with all the little bits and bobs in it :man_shrugging:

**Review by nkpng2k — APPROVED** [review]
@MartinBarus thanks for this PR. It does bring up a good question. Since this is primarily a java repo at the moment. I think it is a good time maybe to revisit this issue from some time back: https://github.com/h2oai/dai-deployment-templates/issues/129

I would maybe say for now to make the directory as `dai-deployment-templates/python/mlflow` simply to avoid confusion? wdyt?

**Comment by jakubhava** [comment]
Closing as stale

**Pull request closed without merging**